### PR TITLE
Document how to try storage-ng in installation

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,19 @@ rpm -e --nodeps libstorage6 libstorage-ruby libstorage-python libstorage-devel l
...
+## Installation of System with Storage-ng
+
+If you are brave enough to install openSUSE Tumbleweed using this module, despite
+the facts that **is not supported and can produce data loss**, you can download the latest ISO
+image from [build service](http://download.opensuse.org/repositories/YaST:/storage-ng/images/iso/).
+
+Known issues:
+
+- not all functionality implemented ( see Status below )
+- disk have to be at least 25GB big
```
Comment (ancorgs) [inline README.md]: This is only true if you use Btrfs and a separate home.

Comment (jreidinger) [inline README.md]: and for other scenarious? is it known?

Comment (jreidinger) [inline README.md]: and for tumbleweed it is default, so it is for default setup, right?

File: README.md [hunk 2]
```diff
@@ -36,6 +36,19 @@ rpm -e --nodeps libstorage6 libstorage-ruby libstorage-python libstorage-devel l
...
 
+## Installation of System with Storage-ng
+
+If you are brave enough to install openSUSE Tumbleweed using this module, despite
+the facts that **is not supported and can produce data loss**, you can download the latest ISO
+image from [build service](http://download.opensuse.org/repositories/YaST:/storage-ng/images/iso/).
+
+Known issues:
+
+- not all functionality implemented ( see Status below )
```
Comment (ancorgs) [inline README.md]: Rubocop would complain here about surplus spaces within parenthesis ;)

File: README.md [hunk 3]
```diff
@@ -36,6 +36,19 @@ rpm -e --nodeps libstorage6 libstorage-ruby libstorage-python libstorage-devel l
...
+- for default setup disk has to be at least 25GB big. Without btrfs and separated home, less is needed.
+- if there are already btrfs with subvolumes, it have to be wiped before installation
```
Comment (ancorgs) [inline README.md]: it has

**Pull request merged**